frontend/package.json

```diff
@@ -18,12 +18,12 @@
   },
   "devDependencies": {
     "@types/bcrypt": "^5.0.0",
-    "@types/node": "17.0.31",
+    "@types/node": "17.0.30",
     "@types/react": "18.0.8",
     "@types/validator": "^13.7.2",
     "cross-env": "^7.0.3",
     "eslint": "8.14.0",
-    "eslint-config-next": "12.1.6",
+    "eslint-config-next": "12.1.5",
```

Comment:
Interesting. The versions got downgraded, can this be undone?

Reply:
I've fixed this. Not sure how it happened. Probably something went wrong while merging